src/routes/safe/components/Apps/index.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react'
 
+import { useLocation } from 'react-router-dom'
+
 import AppFrame from './components/AppFrame'
 import AppsList from './components/AppsList'
 
-import { useLocation } from 'react-router-dom'
+export type TransactionParams = {
```

Comment:
I can see this same type in `src/routes/safe/components/Apps/components/AppFrame.tsx` Line 72

Reply:
A leftover from conflicts 😛 good catch